src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -22,8 +25,9 @@
 
 	@Override
 	public void run() {
-		// TODO Implement.
-		logger.info( "Not yet implemented!" );
+		Configuration.ensureLoaded();
```

Comment:
I think we want at least a simple test for this? Maybe Picocli provides some facilities for testing?

Reply:
Some information on testing picocli apps:

https://github.com/remkop/picocli/issues/367

Let's talk about this create some integration tests.

Reply:
@kema91 created some integration tests using [System Rules](https://stefanbirkner.github.io/system-rules/):

https://github.com/retest/recheck-cli/commit/e19bafcfe43f026a343a3df332f9647151513a69

Looks like a reasonable approach.